package.json

```diff
@@ -1,812 +1,819 @@
 {
-    "name": "vscode-azureappservice",
```

Comment:
Your formatter replaced all the four-space indents with tabs; you'll want to revert that.